dex/testing/eth/create-node.sh

```diff
@@ -80,6 +79,7 @@ DatasetDir = "${NODE_DIR}/.ethash"
 [Node]
 DataDir = "${NODE_DIR}"
 AuthPort = ${AUTHRPC_PORT}
+JWTSecret = "${NODE_DIR}/jwt.hex"
```

Comment:
geth documentation is not great

I do know that if you try to start a node with an account unlocked and http on, it panics. Will look into it, but we do not want to set either --http or --ws.

Reply:
Thanks for posting those links.  This is coming into focus now.

Concepts:
- API **namespaces** (`net`, `eth`, `debug`, `personal`, `engine`, etc)
- API **interfaces** (`--http`, `--ws`, `--authrpc`) e.g. ipc/hostport
- public vs private _namespaces_
- authenticated vs unauthenticated _interfaces_

The namespaces have been classified as either public (e.g. `net`) vs private (`personal`).  That concept has been recently removed and the `Public` deprecated in the exported types as you have pointed out.  https://github.com/ethereum/go-ethereum/pull/25053 https://github.com/ethereum/go-ethereum/pull/25059

The authenticated API appears to have been added because of the `engine` API used for beacon chain communications, which according to the spec "**MUST**" be on an authenticated interface that is also separate from the other API interfaces.  https://github.com/ethereum/execution-apis/blob/main/src/engine/authentication.md